File: jpress-engine/src/main/java/by/ibragimov/jpress/filters/TitleFilter.java
package by.ibragimov.jpress.filters;

import org.springframework.stereotype.Component;

/**
 * TODO: CommentMe!
 *
 * @author Ibragimov Ruslan
 */
@Component
public class TitleFilter implements JPressFilter<String> {

    @Override
    public String filter(String s) {
        return this.getClass().getCanonicalName();
    }
}

File: kpress-engine/src/main/kotlin/io/heapy/kpress/filters/TitleFilter.kt
package io.heapy.kpress.filters

import io.heapy.kpress.api.Filter

/**
 * @author Ruslan Ibragimov
 */
class TitleFilter : Filter<String> {
    override fun tag(): String = "kp_title"

    override fun filter(data: String): String {
        return "$data : Kpress"
    }
}
